### DISCUSSION TITLE
Windows version Design Tab

### DISCUSSION THREAD

**Discussion by terrypen** [meerk40t/meerk40t#2475] [Q&A] [answered]
The Design Tab is missing on my Windows version I downloaded today! Or am I doing something wrong?
Terry

**Comment by jpirnay** [comment, marked as answer]
Sorry for the late reply - please download the latest version again. Starting from version 0.9 the design tab is no longer in the top section but a toolbar of its own on the edge of the screen. Please make sure it's activated in the panes menu. 
You can as well mirror some of the tools in the top section by customizing it in preferences
![grafik](https://github.com/meerk40t/meerk40t/assets/2670784/eaaeedb6-387a-455f-88db-e0a4d80b6ada)
